Fixed some typos in Mastering Plone 6 > 4, 11, 12 (#733)
plone/training: docs/mastering-plone/case.md

By 2050 human civilisation has reached the stars and each year the conference is held on a different planet.

> The conference-website we want to create should be reusable in the following years so some things should be configurable.

Reviewer: You might see an overabundance of hyphenated words, such as this. Germanic languages do this a lot, but in English it is less common.

```suggestion
The conference website we want to create should be reusable in the following years, so some things should be configurable.
```

- Create a sponsor contenttype to manage sponsors
- Store non-visible information about the sponsor in the sponsor-type
- Display sponsors on all pages sorted by level

> - Allow some available to be configurable by a admin in a controlpanel

Reviewer: This sentence does not make sense. It is missing a word after "available", or perhaps the intent is "availability"? I am not sure. Would you please look elsewhere in this Training to infer what it should be?

```suggestion
- Allow some available to be configurable by an admin in a control panel
```

> - Allow some available to be configurable by an admin in a control panel

Reviewer 2: ```suggestion
- Allow some settings to be configurable by an admin in a control panel
```
